wikitongues/wikitongues/config/load_configs.py

```diff
@@ -43,3 +48,26 @@ def load_configs():
         # nothing overridden. return the default config settings
         print("Using default configuration")
         return default_config
+
+def load_airtable_datastores(config):
```

Comment:
Could you make this two separate functions, one for each table

Reply:
yep